src/components/Login/Login.js

```diff
@@ -12,48 +11,35 @@ export const LoginComponent = () => {
     let mail = target.mail.value;
     let password = target.password.value;
     validarUsuario(mail, password);
-  }
+  };
 
   const validarUsuario = async (mail, password) => {
-    const url = `${dominio}/login`;
-    const response = await fetch(url, {
-        method: 'POST',
-        headers: {
-            'Access-Control-Allow-Origin': '*',
-            'Content-Type': 'application/json'
-        },
-        body: `{
-            "mail": "${mail}",
-            "password": "${password}"
-        }`,
-    });
-    const data = await response.json();
-    console.log(data);
-    if(data.status === 'FAILED'){
+    const data = await validateLogin(mail, password);
+    if (data.status === "FAILED") {
       alert("datos erroneos");
-    }else{
-      localStorage.setItem("accessToken", data.data.accessToken);
-      navigate('/home');
+    } else {
+      localStorage.setItem("accessToken", data.accessToken);
```

Comment:
create a helpers functions for localStorage
something like this
```javascript
const getLocalStorageItem = key => {
    return window.localStorage.getItem(key);
};

const setLocalStorageItem = (key, value) => {
    window.localStorage.setItem(key, value);
};

const removeLocalStorageItem = key => {
    window.localStorage.removeItem(key);
};

export { getLocalStorageItem, setLocalStorageItem, removeLocalStorageItem };
```
and then import it and use it as you like

you could create a folder helpers -> localStorage.helpers.js